src/LinkDotNet.Blog.Web/Features/Archive/ArchivePage.razor

```diff
@@ -31,7 +32,7 @@
 								{
 									<li class="list-group-item">
 										<a href="/blogPost/@blogPost.Id/@blogPost.Slug" class="text-decoration-none">@blogPost.Title</a>
-										<span class="text-muted small ps-2">(@blogPost.UpdatedDate.ToString("MMM dd, yyyy"))</span>
+										<span class="text-muted small ps-2">(@blogPost.UpdatedDate.ToString("MMM dd, yyyy", CultureInfo.CurrentCulture))</span>
```

Comment:
Wouldn't `ToShortDateString` be the better fit here as well?

Reply:
If you want to keep the format (Sep 04, 2024) then we would want this, `ToShortDateString` is in the format of 4/9/2024 or 9/4/2024
![image](https://github.com/user-attachments/assets/7a9fca3c-563c-44a7-92bd-81aa1a79b716)
![image](https://github.com/user-attachments/assets/57c9e09c-7dcc-46a3-a465-daf550672b6f)

Reply:
Fair - let's keep it that way.